common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +53,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    invoker: Option[String] = None)
```

Comment:
I put the invoker to the Message, because I moved connectivities (Kafka) from the LoadBalancerService to the InvokerPool.
To transfer the invokername, that the LoadBalancer has chosen to the invokerPool, I use this field.